capitals.rb

```diff
@@ -150,4 +153,50 @@
 }, {
     name: "Wyoming",
     capital: "Cheyenne"
-}]
+}].shuffle
+
+# states arranged in non alphabetical order
+
+
+# Initialize new keys to sort the num time of correct and wrong answers
+states[:num_correct]
+states[:num_wrong]
```

Comment:
set these to 0. if we're initializing a counting variable, its initial value should be 0